launch/line_segment_detector.launch

```diff
@@ -0,0 +1,34 @@
+<launch>
+  <arg name="node_name" default="line_segment_detector" />
+
+  <arg name="image" default="image" doc="The image topic. Should be remapped to the name of the real image topic." />
+
+  <arg name="use_camera_info" default="false" doc="Indicates that the camera_info topic should be subscribed to to get the default input_frame_id. Otherwise the frame from the image message will be used." />
+  <arg name="debug_view" default="true" doc="Specify whether the node displays a window to show edge image" />
+
+  <arg name="lsd_refine_type" default="2" doc="Line Segment Detector Modes"/>
```

Comment:
please add list of types : see https://github.com/ros-perception/opencv_apps/blob/indigo/launch/edge_detection.launch#L9